src/components/search/SearchResults.tsx

```diff
@@ -0,0 +1,88 @@
+import Fuse from "fuse.js";
+import { useQuery } from "@tanstack/react-query";
+import { useEffect, useState } from "react";
+import PostCard from "../PostCard";
+import LoadingSpinner from "../ui/LoadingSpinner";
+
+export type ResultObj = {
+  slug: string;
+  data: {
+    title: string;
+    description: string;
+    date: string;
+  };
+};
+
+type SearchResult = {
+  item: ResultObj;
+  refIndex: number;
+  score: number;
+};
+
+const DEFAULT_NUM = 5;
```

Comment:
nit: could we get more specific with this variable name? Like `DEFAULT_NUM_RESULTS_SHOWN` or something like that?

Reply:
Yeah, I like that!